### PR TITLE
Run `cargo fmt`

### PR DESCRIPTION
**Pull request by sergey-melnychuk**
https://github.com/rust-lang/rustfmt#checking-style-on-a-ci-server

### CODE REVIEW

File: whisper_tests/tests/update.rs [hunk 1]
```diff
@@ -51,75 +54,147 @@ fn test_update_snapshot() -> Result<(), Error> {
...
+            .u32(0)
+            .f64(0.0)
+            .u32(0)
+            .f64(0.0)
+            .u32(0)
+            .f64(0.0)
+            .u32(0)
+            .f64(0.0)
+            .u32(0)
+            .f64(0.0)
```
Comment (andy128k) [inline whisper_tests/tests/update.rs]: This is an example why I don't like autoformatting

**Comment by sergey-melnychuk** [comment]
@AnderEnder , @Deitrl , friendly ping :)

**Comment by AnderEnder** [comment]
@andy128k and @sergey-melnychuk, squash and merge?

**Comment by sergey-melnychuk** [comment]
OK, we have a quorum. Merging.

**Pull request merged**